Gemfile

```diff
@@ -44,7 +44,7 @@ gem 'omniauth-zooniverse'
 gem 'responders'
 gem 'listen', '>= 3.0.5', '< 3.2'
 gem 'rest-client', '> 2.0'
-gem 'aws-sdk', '~> 2.4.4'
+gem 'aws-sdk-s3'
```

Comment:
AWS has split up their gems, you can still load 'aws-sdk' but that depends on ALL of the services, and we only need S3.